app.js

```diff
@@ -11,29 +11,29 @@ const app = express();
 app.set('view engine', 'ejs');
 
 // Public Folder
-app.use(express.static(path.join(__dirname,'public')));
+app.use(express.static(path.join(__dirname, 'public')));
 app.use(express.json())
-app.use(express.urlencoded({extended:true}))
+app.use(express.urlencoded({ extended: true }))
 
 
-
-app.get('/', (req, res) => res.render('index',{
-  files:[]
+app.get('/', (req, res) => res.render('index', {
+  files: []
```

Comment:
Can we do something else?
`app.get('/', (req, res) => res.render('index', {
  files: []
)}`

can you do this [here](https://github.com/coder-mars/nodeuploads/blob/93487200896f8176935f4925d5b9576e08a0bafa/app.js#L25)